### DISCUSSION TITLE
How to make Georgia continue album from selected song

### DISCUSSION THREAD

**Discussion by chromalife** [TT-ReBORN/Georgia-ReBORN#10] [Q&A] [answered]
Hello,

I have a question about this skin and it's functionality. I have installed it, and I have noticed that if I want to continue an album from a certain song, the player does not do this. Example, if I select track three, the player will only play track 3 and not play track 4 and onward. I figure that I'd have to select the album, then hit the skip button to the track I wish to hear, which can be annoying. Any thoughts? Thanks!

**Comment by TT-ReBORN** [comment]
Hi,

 I don't know what you did but it works as it should. Maybe you have hit the shuffle button and randomized your playlist...
Just drag any album to the playlist and it should work without problems. Make sure the whole album is in a folder and you have
tagged it correctly ( correct artist name, album title, track numbers and song titles ). 

-TT

**Comment by Groxxxy** [comment]
Hi, I experienced this too and wanted to post here about it. I got carried away with editing either:

georgia-reborn-config.jsonc

or

georgia-reborn-main.js (to remove the rating functionality)

and afterwards songs would play after one another. I don't know at what point my tinkering fixed the poster's original issue on my side but they have a valid issue.

If you install a fresh portable instance of Foobar and apply Georgia-Reborn you will notice the next song does not start.

Thanks.

**Reply by TT-ReBORN** [reply]
There is nothing wrong when installing a fresh portable foobar with Georgia-ReBORN.
The songs play just normal in sequential order if you add an album or just different songs...
You two were the first to report this and if you modify any Georgia-ReBORN files I can't give support...
I don't know what you're modifying or what other user-components you've installed, but if you use the original files
it should work out of the box. I have never encountered this since I'm using my own theme the last two years...

Can you please tell me exactly how to reproduce this?

**Reply by TT-ReBORN** [reply]
One thing that comes to my mind is that most HDD's have an integrated sleep mode. 
It caches one or two songs in memory and if you manually click at the beginning of the song with your mouse cursor in the progress bar and you repeat it this few times, after about 15-20 minutes most HDD's go into sleep mode. When the next new song wants to play, it needs to wait a few seconds until the HDD will wake up from sleep mode and then it will continue. But this is normal...

**Comment by Groxxxy** [comment]
No problem, it seems my tinkering did not fix the problem after all but instead I started using Foobar with Reborn differently to allow songs to play sequentially. 

What's happening is if you choose a song from library structure view it will only play that one song.

If however you open the album from library structure view and click library tab again to move to album view songs will play in sequence.

Basically selecting one song in library structure view plays that one song; selecting the entire album AND then selecting a song will play them in sequence.

Please see that selecting only one track plays it:
![pQadhWc1GB](https://user-images.githubusercontent.com/14025275/146106796-5b282129-9a12-495e-8d29-55d70f602740.gif)


Selecting the entire album and switching to album view ✔️:
![rfpOVK6xKW](https://user-images.githubusercontent.com/14025275/146106981-0a7ba8ef-6acf-4eb6-b784-ff552f05a9f0.gif)

**Comment by TT-ReBORN** [comment, marked as answer]
That's because you're only adding one song to the library view playlist. 
Double click on one song and go to playlist ( if library panel is active, click again on the library top menu button to go back to the playlist ),
only one song has been added to the library view playlist. If you double click on an entire album, the whole album will be added to the playlist. If you double click, it will always replace the clicked albums/songs in the library view playlist. 
If you don't want to replace the library playlist view just right click for context menu and select > Add to current playlist.
Or if you make a new playlist and the library view playlist is not active but another, it will add songs/albums to the current active
playlist.

You should also play around the playlist manager to get a better understanding how it works.

-TT
